docs/ci-operations-group.md

```diff
@@ -0,0 +1,140 @@
+# KubeVirt CI Operations Group
+
+
+## Goal of this document
```

Comment:
Would it be useful to also include information on who to contact or which meeting to attend if someone wants to be involved?

Reply:
same as above i guess?

Reply:
Sorry, somehow that doubled up.